service/nodemailer.js

```diff
@@ -0,0 +1,32 @@
+const nodemailer = require('nodemailer');
+require('dotenv').config();
+
+const config = {
+  host: process.env.MAILTRAP_SERW,
+  port: 2525,
+  auth: {
+    user: process.env.MAILTRAP_USER,
+    pass: process.env.MAILTRAP_PASS,
+  },
+};
+
+const transporter = nodemailer.createTransport(config);
+const emailOptions = (reciver, verificationToken) => {
+  const email = {
+    from: 'charyszek@interia.pl',
+    to: `${reciver}`,
```

Comment:
?
```
to: reciver,
```

Reply:
trochę się zaplątałem. Chciałem mieć pewność że to jest string.. ale prawda jest taka że maile też są zapisane jako stringi więc niepotrzebne dublowanie.